lldb/test/API/lang/swift/async/async_fnargs/TestSwiftAsyncFnArgs.py

```diff
@@ -14,6 +14,18 @@ class TestSwiftAsyncFnArgs(lldbtest.TestBase):
     def test(self):
         """Test function arguments in async functions"""
         self.build()
+        self.do_test()
+
+    @swiftTest
+    @skipIf(oslist=['windows', 'linux'])
+    def test_backdeploy(self):
+        """Test function arguments in async functions"""
+        self.build(dictionary={
+            'TARGET_SWIFTFLAGS': f'-target {self.getArchitecture()}-apple-macos11'
```

Comment:
This is the first time I'm seeing these nifty format string literals. Are they supported by all python versions that LLDB can build under?

Reply:
I'll get rid of these. I forgot that f-strings are Python 3.6+, not all versions of Python 3.